examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+  * [Create the example namespace](#create-the-example-namespace)
+  * [Get Coherence Docker image](#get-coherence-docker-image)
+  * [Create a secret](#create-a-secret)
+  * [Clone the GitHub repository](#clone-the-github-repository)
+  * [Install the Coherence Operator](#install-the-coherence-operator)
+  * [Port Forward Grafana and Kibana ports](#port-forward-grafana-and-kibana-ports)
+* [Run the Examples](#run-the-examples)
+  * [Example 1 - Coherence cluster only](#example-1---coherence-cluster-only)
+  * [Example 2 - Adding a Proxy role ](#example-2---adding-a-proxy-role) 
+  * [Example 3 - Adding a User application role](#example-3---adding-a-user-application-role)
+  * [Example 4 - Enabling Persistence](#example-4---enabling-persistence)
+* [Cleaning Up](#cleaning-up)  
+
+# Prerequisites
+
+## Coherence Operator Quick Start
+
+Ensure you have followed all the [Quick Start Guide](https://oracle.github.io/coherence-operator/docs-unstable/2.0.0-1910091023/#/about/03_quickstart) including the
+prerequisites and have been able to successfully install the Coherence Operator and a Coherence Cluster.
+
+## JDK and Maven versions
+
+Ensure you have the following software installed:
+
+* [Java 8+ JDK](http://jdk.java.net/)
+* [Maven](https://maven.apache.org) version 3.5+
+* [Docker](https://docs.docker.com/install/) version 17.03+.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.11.3+.
+* Access to a Kubernetes v1.11.3+ cluster.
+
+> Note: You can use a later version of Java, for example, JDK11, as the
+>`maven.compiler.source` and `target` are set to JDK 8 in the example `pom.xml` files.
+
+> Note: Ensure that your local Kubernetes is enabled. If you are running Docker using Docker Desktop, select Enable Kubernetes in the Settings menu.
+
+## Install Coherence into local Maven repository
+
+1. Download and install Oracle Coherence 12.2.1.4 from [Oracle Technology Network](https://www.oracle.com/technetwork/middleware/coherence/downloads/index.html).
+
+1. Ensure that the `COHERENCE_HOME` environment variable is set to point to the `coherence` directory under your install location containing the bin, lib, and doc directories. This is required only for the Maven `install-file` commands.
+
+1. Install Coherence into your local Maven repository:
+
+   ```bash
+   mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence.jar   \
+       -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence/12.2.1/coherence.12.2.1.pom
+  
+   mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-metrics.jar \
+       -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-metrics/12.2.1/coherence-metrics.12.2.1.pom
+   ```
+
+## Create the example namespace
+
+You need to create the namespace for the first time to run any of the examples. Create your target namespace:
+
+```bash
+kubectl create namespace coherence-example-ns
```

Comment:
Do we need `-ns` on the end, it's obviously a namespace

Reply:
have fixed this